waspc/testEnv.hs

```diff
@@ -19,6 +19,7 @@ import Data.Aeson
 import Data.Aeson.Encode.Pretty
 import Data.ByteString.Lazy.Char8 as L
 
+import Parser.Common (runWaspParser)
```

Comment:
Why this has to be added now and it was not added before?

Reply:
I found it handy for testing a specific parser interactively. I wasn't doing it before so I didn't introduce it earlier.